model.py

```diff
@@ -8630,6 +8648,48 @@ class Collection(Base):
         backref="collections"
     )
 
+    catalog = relationship(
+        "Identifier", secondary=lambda: collections_identifiers,
+        backref="collections"
+    )
+
+    @property
+    def metadata_identifier(self):
+        """Identifier based on collection details that uniquely represents
+        this Collection on the metadata wrangler. This identifier is
+        composed of the Collection protocol and account identifier.
+
+        In the metadata wrangler, this identifier is used as the unique
+        name of the collection.
+        """
+        account_id_column = self.UNIQUE_IDENTIFIER_BY_PROTOCOL.get(self.protocol)
+        account_id = getattr(self, account_id_column.name)
```

Comment:
If the collection has a parent ID, its metadata identifier needs to include the metadata identifier for its parent. So an Overdrive Advantage account might have a metadata Id like `MY LIBRARY / 27`.